basics/keeping-up-to-date/_index.md

```diff
@@ -9,87 +9,101 @@ showOnHomepage: true
 
-Keeping PrestaShop up-to-date can be done via different methods. Choose the best update method depending on your needs.
+- **New Features:** Access the latest features and enhancements for an improved experience.
+- **Design & UX Improvements:** Enjoy updated themes and modernized user interfaces.
+- **Security Enhancements:** Protect your store with the latest security patches and performance updates.
+- **Technical Compatibility:** Ensure compatibility with the latest technologies and updates.
+- **Ecosystem Growth:** Take advantage of new modules, themes, and their ongoing updates.
+- **Stability & Reliability:** Benefit from bug fixes and patches that enhance the stability of your store.
+- **Official Support:** Stay within the scope of officially supported PrestaShop versions and services.
 
-### In-place Upgrade
+## Update vs Migration: two distinct processes
 
-![Upgrade schema](img/upgrade-schema.png)
+Keeping PrestaShop up-to-date can be achieved through different methods. Choose the most suitable update process based on your specific needs.
+
+### Updating PrestaShop
 
-Upgrading a shop is the recommended method when you want to get your shop up-to-date without switching to the next major version.
+![Upgrade schema](img/upgrade-schema.png)
 
-It does not require any additional tools to run if you follow the manual process, the most recent release package will be enough.
+Updating your store is the recommended approach when you want to stay up to date without switching to the next major version.
 
 #### Impact on existing data
 
-As long as you stay on the same major version (ex. 1.7.1 >> 1.7.2, 8.0 >> 8.1 upgrade), we make sure that the available features remain the same.
-This means that your current theme and all your modules should continue to work as before, and no functionality or data will be lost during the upgrade, even if the database structure may change.
+As long as you remain within the same major version (e.g., 8.0 → 8.1, 9.0 → 9.1), PrestaShop ensures that all existing features remain intact. This means:
 
-This can be explained by the [semantic versioning](https://semver.org/) we follow, that forbids any compatibility-breaking change in the core, such as removing a feature or modifying our APIs in versions other than major.
+- Your current theme and modules will continue to function as expected.
+- No data or functionalities will be lost, even if database structures are modified.
 
-Note that once an upgrade has started, there is no way to rollback the changes. The only solution you have is restoring the backup you made before.
+This stability is guaranteed by the <a href="https://semver.org/" target="_blank">semantic versioning</a> system we follow, which prevents any compatibility-breaking changes in minor and patch updates. Core features, APIs, and functionality remain unchanged outside of major releases.
```

Comment:
```suggestion
This stability is guided by the <a href="https://semver.org/" target="_blank">semantic versioning</a> system we follow, which aims to prevent compatibility-breaking changes in minor and patch updates. However, in rare cases, critical bug fixes or security patches may require adjustments that could affect compatibility. Outside of major releases, core features, APIs, and functionality generally remain stable.
```